pullrequest.rst

```diff
@@ -93,6 +93,85 @@ command (after any successful build of Python)::
    python.bat Tools/scripts/patchcheck.py
 
 
+.. _pullrequest-quickguide:
+
+Quick Guide
+'''''''''''
+
+`Clear communication`_ is key to contributing to any project, especially an
+`Open Source`_ project like Python.
+
+Here is a quick overview of you can contribute to CPython on GitHub:
+
+1.  If an Issue doesn't exist, `create an Issue`_ that describes your change
+
+2.  Prepare your system by :ref:`setup`
+
+3.  Fork `CPython`_ on GitHub (using the Fork button in the upper-right on GitHub)
+
+4.  Start :ref:`compiling` to build Python
+
+5.  Begin :ref:`runtests` after you have built Python
+
+6.  Set your Git :ref:`remote-configuration` to add an "upstream" remote (using SSH, or you can `use HTTPS`_)
+
+7.  Create :ref:`committing-active-branches` in Git where you can work on changes
```

Comment:
I think the reference to `committing-active-branches` is incorrect here. 
Currently it links to the section that explains how to list the branches (`git branch` command.)
It's better to refer to line 153 below (about `git checkout -b BRANCH_NAME`) which is the key step.